src/main/landunit_varcon.F90

```diff
@@ -18,9 +18,10 @@ module landunit_varcon
 
   integer, parameter, public :: istsoil    = 1  !soil         landunit type (natural vegetation)
   integer, parameter, public :: istcrop    = 2  !crop         landunit type
-  ! Landunit 3 currently unused (used to be non-multiple elevation class glacier type:
-  ! istice, and landunit 4 was istice_mec; now they are combined into a single landunit
-  ! type, 4)
+  integer, parameter, public :: istocn     = 3  !ocean        landunit type
+  ! Landunit 3 used to be non-multiple elevation class glacier type, istice
+  ! Landunit 4 was istice_mec; now landunit 4 and old landunit 3 are combined
+  ! into landunit 4
```

Comment:
Reviewing with @ekluzek:

Remove these comments that refer to past code history.